modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/js/message_boards/MBPortlet.js

```diff
@@ -246,42 +245,6 @@ class MBPortlet {
 		}
 	}
 
-	/**
-	 * Updates the attachments to include the checked attachments.
-	 *
-	 */
-
-	_updateMultipleMBMessageAttachments() {
-		const namespace = this._namespace;
-
-		const selectedFileNameContainer = document.getElementById(
-			`${namespace}selectedFileNameContainer`
-		);
-
-		if (selectedFileNameContainer) {
-			const inputName = `${namespace}selectUploadedFile`;
-
-			const input = [].slice.call(
-				this.rootNode.querySelectorAll(
-					`input[name=${inputName}]:checked`
-				)
-			);
```

Comment:
@Mikellorza This doesn't look correct. Previously, the `selectedFileName` parameter would be filled with those file names that were checked; by removing this code and using the `selectUploadedFile` directly, you're always uploading everything, no matter the selection status of the checkbox.

Reply:
We don't need that code already because the select file component do this for us. I send a video for you through slack because I can't upload the video because of the size here.

Reply:
I see. Sounds good.

Reply:
Can I get the video to because I don't understand?

Reply:
I left the video in the description @boton